docs/src/guide/emits.md

```diff
@@ -2,38 +2,52 @@
 
 You can access up-to-date data in the operations you perform for the tree with the `emit`'s we use in the package.
 
+### `nodeClick`
+
+This emit is triggered when a node is clicked or selected.
+
+#### Usage
+
+```vue
+<Tree :nodes="data" @nodeClick="nodeClick" />
```

Comment:
It should be updated to `@nodeClick="onNodeClick"`

Reply:
Fixed. 👍🏻